inline-c/examples/gsl-ode.hs

```diff
@@ -94,7 +94,7 @@ solveOde
   -> Either String (V.Vector Double)
   -- ^ Solution at end point, or error.
 solveOde fun x0 f0 xend =
-  coerce $ solveOdeC (coerce fun) (coerce x0) (coerce f0) (coerce xend)
+  unsafeCoerce $ solveOdeC (unsafeCoerce fun) (unsafeCoerce x0) (unsafeCoerce f0) (unsafeCoerce xend)
```

Comment:
It seems that not only `instance Coercible (Vector CDouble) (Vector Double)` but also `instance Coercible CDouble Double` are not defined.
So this update is needed.
This is related to https://github.com/fpco/inline-c/issues/146 .